src/sentry/lang/javascript/processing.py

```diff
@@ -142,21 +142,41 @@ def map_symbolicator_process_js_errors(errors):
 
 
 def _handles_frame(frame, data):
-    if not frame:
-        return False
```

Comment:
I mean, I just copied this code from someplace else, not sure if we actually need to be defensive here.